doc/lsst.pipe.base/command-line-task-parallel-howto.rst

```diff
@@ -0,0 +1,97 @@
+.. _command-line-task-parallel-howto:
+
+###########################################
+Parallel processing with command-line tasks
+###########################################
+
+:ref:`Command-line tasks <using-command-line-tasks>` can operate in parallel.
+Generally if a command-line task is run against several :ref:`data IDs <command-line-task-dataid-howto>`, each data ID can be processed in parallel.
+Command-line tasks use two different categories of parallel processing: single-node parallel processing (:option:`-j`) and distributed processing (with the ``pipe_drivers`` and ``ctrl_pool`` packages).
+This page describes how to use these parallel processing features.
+
```

Comment:
A bit more about what situations might warrant single-node parallel processing vs. distributed processing would be nice here.